akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Protocol.scala

```diff
@@ -191,7 +191,9 @@ object Http2Protocol {
     def byId(id: Int): SettingIdentifier = All(id - 1)
   }
 
-  sealed abstract class ErrorCode(val id: Int) extends Product
+  sealed abstract class ErrorCode(val id: Int) extends Product {
+    def name = productPrefix
```

Comment:
I used it in some point during testing to get better error messages. I see it's not used anymore, removing